wiki/People/Staff_Log/2019/en.md

```diff
@@ -65,3 +65,30 @@ All users listed below were added on 2019-02-16.
 ## March
 
 - Moved [Mir](https://osu.ppy.sh/users/8688812) to the Quality Assurance Team (2019-03-23)
+
+## April
+
+- Moved [Fycho](https://osu.ppy.sh/users/1876867) to the Global Moderation Team (2019-04-10)
```

Comment:
Missing Ascendance being added to the QAT.

`- Added [Ascendance](https://osu.ppy.sh/users/2931883) to the Quality Assurance Team (2019-04-10)`